app/components/billing-invoices.js

```diff
@@ -0,0 +1,25 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+
+  invoiceYears: computed('invoices.@each.{createdAt}', function () {
+    const invoiceYears = this.invoices.map(invoice => invoice.createdAt.getFullYear());
+    const distinctInvoiceYears = new Set(invoiceYears);
```

Comment:
There's a handy array prototype extension method in Ember for this: https://api.emberjs.com/ember/3.11/classes/EmberArray/methods/uniq?anchor=uniq

This would allow to shorten the implementation:
```js
return invoiceYears.uniq().sort((a, b) => b - a);
```